plugins/animation_to_java.js

```diff
@@ -0,0 +1,84 @@
+(function () {
+  var menuButton;
+
+  Plugin.register("animation_to_java", {
+    title: "Animation to Java Converter",
+    author: "MG",
+    description:
+      "Converts Blockbench animations to Java code for the new 1.19 keyframe system",
+    icon: "fa-cube",
+    version: "1.0.0",
+    variant: "both",
+    tags: ["Minecraft: Java Edition"],
+    onload() {
+      menuButton = new Action("export_to_java", {
+        name: "Export Animations to Java",
+        description: "Exports animations to Java code",
+        icon: "fa-file-export",
```

Comment:
I would recommend to add a condition here so that the action doesn't show up in all formats.
Something like:
`condition: () => Format.animation_mode`

Reply:
Add the condition for the action, not for the entire plugin. Otherwise it will also not work when switching to formats with animation mode later.